.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "riscv-binutils-gdb"]
+	path = riscv-binutils-gdb
+	url = https://github.com/riscv/riscv-binutils-gdb.git
```

Comment:
It's actually pretty slick, it forms a relative URL not a local path.  So if you clone `http://github.com/riscv/riscv-gnu-toolchain.git` and it references `url = ../riscv-binutils-gdb.git` it will build `http://github.com/riscv/riscv-binutils-gdb.git` as the submodule URL.